test/conformance/api/v1beta1/errorcondition_test.go

```diff
@@ -60,7 +60,7 @@ func TestContainerErrorMsg(t *testing.T) {
 	// Specify an invalid image path
 	// A valid DockerRepo is still needed, otherwise will get UNAUTHORIZED instead of container missing error
 	t.Logf("Creating a new Service %s", names.Service)
-	svc, err := createService(t, clients, names, 2)
+	svc, err := v1b1test.CreateService(t, clients, names)
```

Comment:
```suggestion
	svc, err := v1b1test.CreateService(t, clients, names, WithRevisionTimeoutSeconds(2))
```

Reply:
Why is that option even needed for the test?

Reply:
hmm right, it seems this is not what this test is testing.